agent/catalog_endpoint_test.go

```diff
@@ -8,9 +8,11 @@ import (
 	"time"
 
 	"github.com/hashicorp/consul/agent/structs"
+   "github.com/hashicorp/consul/types"
 	"github.com/hashicorp/consul/testutil/retry"
 	"github.com/hashicorp/serf/coordinate"
 	"github.com/stretchr/testify/assert"
+   "github.com/stretchr/testify/require"
```

Comment:
Nit: Is this an artifact of the GitHub diff viewer or are these not tabbed/formatted properly?

Reply:
Right you are, looks like something happened to my editor plugin to run go fmt on save.